### ISSUE TITLE
How to improve the output vedio resolution in colab? i want 1080 X 1920

### ISSUE THREAD

**Issue by MiaoJiawei97** [anothermartz/Easy-Wav2Lip#57]
i use colab to generate vedio, my input vedio is 1080 X 1920, but out vedio is 540 X 960，i want get a config to change the output vedio file resolution, hope it can output 1080 X 1920 vedio, it will be much clear。

**Comment by anothermartz** [comment]
Is "output_height" set to "full resolution"?

It can be set to "half resolution" which seems to be what's happening here.

**Comment by MiaoJiawei97** [comment]
yes, i always set output_height to full resolution, and the output file is 540 X 960, it's strange

**Comment by MiaoJiawei97** [comment]
and i don't know what happened, from about 3 hours ago, my output vedio seem very strange, the face appears in the corner of the video.
